### DISCUSSION TITLE
The dify tool returns a list of options, and the user can select multiple options.

### DISCUSSION THREAD

**Discussion by jaxonister** [langgenius/dify#8013] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

The dify tool returns a list of options, and the user can select multiple options to interact

**Comment by sanpatricky** [comment]
please consider this feature request, it's very useful

**Comment by theobouwman** [comment]
Is this now possible in dify?
